src/dbnode/storage/types.go

```diff
@@ -1316,4 +1372,4 @@ type NamespaceHooks interface {
 	OnCreatedNamespace(Namespace, GetNamespaceFn) error
 }
 
-type GetNamespaceFn func (id ident.ID) (Namespace, bool)
+type GetNamespaceFn func(id ident.ID) (Namespace, bool)
```

Comment:
I know it's not related to this change but noticed this needs an exported comment?